src/lib.rs

```diff
@@ -1,19 +1,29 @@
 //! `log` compatible logger to the `HiLog` logging system on OpenHarmony
 //!
-//! This crate is in its very early stages and still under development.
-//! It's partially based on [`env_logger`], in particular the filtering
-//! is compatible with [`env_logger`].
+//! This crate is in its very early stages and still under development,
+//! it was partially inspired by [`env_logger`].
+//! 
+//! ## Features
+//! - Permits filtering based on the [`env_filter`] spec via the crate.
+//! - Permits dynamic replacement of filters in an atomic manner, making changes
+//!  immediately available to subsequent invocations on other threads without
```

Comment:
```suggestion
//!  available to subsequent invocations on other threads without
```

I would remove the word immediately here - I think the description in the PR was better.

Reply:
I agree, i think that can be misunderstood to mean that it's synchronized across threads between loading and deref'ing.